doc/whats_new.rst

```diff
@@ -20,6 +20,7 @@ BUG
 API
 ~~~
 
+    - Added logscale epoch plotting option via `logscale=True` in :func:`mne.viz.plot_epochs_image`, by `Sagun Pai`_
```

Comment:
yes remove the negative values and have the 0 from stim channels map to
np.nan after the log so you it's rendered as transparent